desktop/components/react/utils/render_layout.js

```diff
@@ -0,0 +1,36 @@
+import renderTemplate from 'desktop/components/react/utils/render_template'
```

Comment:
Yeeeaaaah! This is even better than I'd imagined!

Now I feel like I did a horrible job explaining my idea originally, LOL.

Question -- what happens if you pass something that's neither type in? Should we log an error so that the caller can be alerted they did something wrong?

Also, curious whether there's a robust way to test for a component. Perhaps by presence of `propTypes` (even though the `auctions2` one doesn't currently have them)?